Cargo.toml

```
@@ -107,15 +107,15 @@ tap = "1.0.1"
 thiserror = "1.0.60"
 dashmap = "5.5.3"
 num_enum = "0.7.2"
-itertools = { version = "0.12.1", default-features = false }
+itertools = { version = "0.13.0", default-features = false }
 portable-atomic = "1.6.0"
 bytemuck = { version = "1.15.0", default-features = false }
 arrayvec = "0.7.4"
 intrusive-collections = "0.9.6"
 cfg-if = "1.0.0"
 either = "1.12.0"
 sys-locale = "0.3.1"
-temporal_rs = "0.0.2"
+temporal_rs = { git = "https://github.com/boa-dev/temporal.git" }
```

```suggestion
temporal_rs = { git = "https://github.com/boa-dev/temporal.git", rev = "61a05fbb7c72353deda72a3df0e6887d65b840d2" }
```

We may want to fix to a hash?